frontend/src/components/associations/marketplace/SidebarStatusSelector.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect } from "react";
+import { SidebarSection } from "../../utils/sidebar/SidebarSection";
+import { CheckboxField } from "../../utils/sidebar/CheckboxField";
+import { useURLState } from "../../../utils/useURLState";
+
+/**
+ * `SidebarStatusSelector` is a sidebar compoenent to search transactions with
+ * a particular status in the sidebar. Its architecture is the same as
+ * `TagSearch`
+ * @param setParams a useState setter to change the query parameters
+ */
+export const SidebarStatusSelector = ({ setParams }) => {
```

Comment:
This name is too generic: polls, library, tutorings… will also need a status selector in their sidebar.

Reply:
But maybe you can transform this component to make it accept any combination of state?